pandas/_libs/tslibs/util.pxd

```diff
@@ -227,3 +236,37 @@ cdef inline bint is_nan(object val):
     is_nan : bool
     """
     return (is_float_object(val) or is_complex_object(val)) and val != val
+
+
+cdef inline const char* get_c_string_buf_and_size(object s,
+                                                  Py_ssize_t *length):
+    """
+    Extract internal char * buffer of unicode or bytes object `s` with
+    getting length of this internal buffer saved in `length`.
+
+    Notes
+    -----
+    Python object owns memory, thus returned char* must not be freed.
+    `length` can be NULL if getting buffer length is not needed.
+
+    Parameters
+    ----------
+    s      : object
```

Comment:
I could be mistaken, but I think we just want one space before the colon

Reply:
Fix count space